packageSourceDeb.nix

```diff
@@ -1,11 +1,13 @@
 # SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
 #
 # SPDX-License-Identifier: MPL-2.0
-{ stdenv, writeTextFile, writeScript }:
+{ stdenv, writeTextFile, writeScript, runCommand }:
 pkgDesc:
 { date, builderInfo }:
 
 let
+  # Otherwise launchpad complains about very old files
+  SOURCE_DATE_EPOCH = 1576846270;
```

Comment:
this variable is now unused.

Reply:
it is not unused, there is a comment which clarifies why I need `SOURCE_DATE_EPOCH`